backend/src/main/java/com/bakdata/conquery/models/datasets/concepts/FrontEndConceptBuilder.java

```diff
@@ -256,7 +259,9 @@ public static FEFilter createFilter(Filter<?> filter) {
 							 .allowDropFile(filter.getAllowDropFile())
 							 .pattern(filter.getPattern())
 							 .defaultValue(filter.getDefaultValue())
+							 .type(filter.getFilterType().getAnnotation(CPSType.class).id())
```

Comment:
Das finde ich nicht gut.  Dann muss jedesmal wenn ein neuer Filter-Typ erstellt wird das Frontend angepasst werden, wo durch es wie bisher zu einem Mismatch zwischen FE und BE kommen kann.

Es ist aus meiner sich sinnvoller Primitives für die FilterValues im Frontend zu erstellen. Ähnlich wie jetzt schon. Generell ist die Gefahr bei mehr feldigen Filtervalues, dass komplexe Validierungen auf frontendseite gewünscht werden.

Reply:
Die CPSType-Id ist das gleiche wie der CPS-Type ich habe nur eine Ebene entfernt

Reply:
Ah stimmt, dann ist das ganze nicht mit meinem Vorschlag kompatibel, denn da gibt es den CPSType nicht mehr

Reply:
Ja das stimmt, das habe ich oben erwähnt. Es bräuchte einen ähnlichen Mechanismus der dann den Deserializer guided